templates/terraform/resource_iam.html.markdown.erb

```diff
@@ -243,14 +243,29 @@ exported:
 
 ## Import
 
-<%= product_ns -%> <%= object.name.downcase -%> IAM resources can be imported using the project, resource identifiers, role and member.
+For all import syntaxes, the "resource in question" can take any of the following forms:
 
+<% import_id_formats(object).each do |id_format| -%>
+* <%= id_format %>
```

Comment:
So we don't support partial import ID formats with resource-type qualifiers anymore, right? i.e. regions/reg/instances/X vs projects/proj/regions/reg/instances/X?

Reply:
We support whatever the parent resource supports, so if we wanted to we could add more into the default formats (or the resource-specific lists), but we don't right now.